### ISSUE TITLE
offline mode support

### ISSUE THREAD

**Issue by kiranpradeep** [Azure/iotedgehubdev#209]
The development computers need not always have an internet connection. Is it possible to let the developer start debugging when offline?

The request is **not** for full-fledged offline support inclusive of resyncing messages or module twin properties. The request is to only let the developer debug while offline with minimal functionalities as possible.

**Comment by adashen** [comment]
@kiranpradeep thanks for the suggestion. However, the edgeHub needs connected internet connection to auth modules. So currently it is not possible to do this.

**Comment by kiranpradeep** [comment]
@adashen EdgeHub allows authentication of modules provided it was able to connect to IoTHub atleast once[[1](https://docs.microsoft.com/en-us/azure/iot-edge/offline-capabilities#how-it-works)]. Is it possible to allow modules to be authenticated provided edgeHub was able to connect to IoTHub atleast once?

**Comment by blackchoey** [comment]
Hi @kiranpradeep If you started running IoT Edge solution using iotedgehubdev with internet connected and then disconnect your network connection, the module could keep working without network connection.

**Comment by kiranpradeep** [comment]
@blackchoey These are different. EdgeHub needs internet connectivity only once in lifetime. But iotedgehubdev needs for every start.

**Comment by blackchoey** [comment]
@kiranpradeep If the EdgeHub container in IoT Edge runtime is deleted, IoT Edge runtime also need to connect to internet in order to start the modules. iotedgehubdev aligns with IoT Edge runtime on this.
The current offline capability in iotedgehubdev enables module keep running if you disconnects from internet, which provides the ability to test your module behavior when lost internet connection.
Please let us know if you need further help. Thanks.

**Comment by konichi3** [comment]
Hi 

Thank you for filing the issue.

We are looking to clean up the reported issues that are obsolete such that our team can focus on the latest issues reported.
As such, we are going to close the issue for now.

If the issue is still blocking or critical to you, please file a new issue and we will do our best to respond as soon as possible.
When you file a new issue, please ensure following additional information are provided.

o	If you believe it’s a bug, provide detail repro steps. 
o	If it’s a feature request, please note as such
o	Expected outcome
o	Is this a blocking issue?

Thank you,
Azure IoT Developer and Device team
